consensus/parlia/parlia.go

```diff
@@ -317,6 +324,153 @@ func (p *Parlia) VerifyHeaders(chain consensus.ChainHeaderReader, headers []*typ
 	return abort, results
 }
 
+func getValidatorBytesFromHeader(header *types.Header, chainConfig *params.ChainConfig, parliaConfig *params.ParliaConfig) []byte {
```

Comment:
A little bit hard to understand, better to have a doc for the layout in different scenario

Reply:
Agree, @NashBC  please add related comments in the code.

Reply:
fixed